src/modules/QtQuick/Item.js

```diff
@@ -41,7 +41,10 @@ function QMLItem(meta) {
     this.dataChanged.connect(this, function(newData) {
         for (var i in newData) {
             var child = newData[i];
-            if (child.hasOwnProperty("parent")) // Seems to be an Item. TODO: Use real inheritance and ask using instanceof.
+
```

Comment:
@ChALkeR this change is not necessary, as Timer has a different parent as mentioned in #287.